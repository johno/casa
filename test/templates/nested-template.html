{{{
  "template": "nested-nested-template"
}}}

<div class="awesome-nested-template">
  <%= content %>
</div>
