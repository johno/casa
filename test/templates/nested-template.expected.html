<div class="awesome-nested-nested-template">
  <div class="awesome-nested-template">

  </div>
</div>
